#input valuesil varunna error aan exception handling
#there have 2 block #try and excecpt

#....................................................................................................
#_______________________________________________________________________________________________________#
#-----------------------------------------------------------------------------------------------------#

a=int(input("enter a number"))
b=int(input("enter a number"))

try:
    print(a/b)#exception chance item

except:
    print("exception occured")#exception vann kazhinjaal ulla item
